doc/admin-guide/files/records.config.en.rst

```diff
@@ -1608,15 +1608,18 @@ Negative Response Caching
    :reloadable:
    :overridable:
 
-   Enables (``1``) or disables (``0``) forcing revalidation of cached documents
-   when |TS| receives a negative (``5xx`` only) response from the origin server. That is,
-   when enabled, |TS| will serve stale when it can not reach the origin server.
+   Negative revalidating allows |TS| to return stale content if revalidation to the origin fails due
+   to network or HTTP errors. If it is enabled, rather than caching the negative response, the current
+   (possibly stale) content is preserved and served. Note this is considered only on a revalidation
```

Comment:
Revalidation can happen on non-stale content. It could be set by a plugin or a user agent sending an IMS, or a `cache.config` setting to force revalidation early.